### ISSUE TITLE
Not compatible with Porting-kit when using 64bit Wine

### ISSUE THREAD

**Issue by Gcenx** [vitor251093/wineskin#16]
I was using the Porting Kit and needed to use a 64bit Wine version.
During creation it complained about the wrapper already running so winetricks failed.
The timeout might need to be extended in Porting Kit for Wine 64bit as they take longer to be created or because of the change between 32bit and 64bit modes.

**Comment by vitor251093** [comment]
That's strange. Usually, PK doesn't work with timeouts. 

Differently from Wineskin, PK "owns" the wrapper running process during its creation, so it knows when the process is killed. It has worked normally so far. Could you please tell me which engine were you using and which Custom Port options have you marked, so I can reproduce the issue?

**Comment by Gcenx** [comment]
I'm not sure how it worked so that was a guess on my part, it works in stock wineskin due to a timeout before closing down the process since wineserver tends to reload.

Oh that might be it then, I have watched a 64bit wrapper during creation and wineserver closes after 32bit prefix files are added, then a new wineserver process opens then 64bit prefix files are added.

I was using WineStaging64Bit3.7 using custom setting for Skyrim wrapper. I was adding additional dotnet versions

**Comment by vitor251093** [comment]
> Oh that might be it then, I have watched a 64bit wrapper during creation and wineserver closes after 32bit prefix files are added, then a new wineserver process opens then 64bit prefix files are added.

I can manage to fix that. Still would need to reproduce it to test the fix thought. 

> I was using WineStaging64Bit3.7 using custom setting for Skyrim wrapper. I was adding additional dotnet versions

You mean you was installing the existing Skyrim port in advanced mode with that engine, or that you were installing a custom port with that engine in order to run Skyrim?

**Comment by Gcenx** [comment]
@vitor251093 I did try to look into this today but PK always replaces the master wrapper now.

Is there a way to disable that so I can check if its working correctly with the current version?

**Comment by vitor251093** [comment]
I can work into that. There are already some changes expecting for a new PK release. I will an option to preferences to enable the use of different versions of the master wrapper, other than the latest as specified by the Wineskin website.

**Comment by vitor251093** [comment]
Alright, just released the PK updated that should solve that problem. Update PK to the latest version (2.9.476) and it should be gone :)

**Comment by Gcenx** [comment]
@vitor251093 I just installed the update, I checked the option but it still deleted the wrapper and gave a funny error;

<img width="624" alt="screen shot 2018-07-15 at 11 47 27 am" src="https://user-images.githubusercontent.com/38226388/42735597-e153d626-8824-11e8-9a12-cdea491d63e1.png">

**Comment by vitor251093** [comment]
Is your master wrapper named in the same pattern as the Wineskin master wrapper? (eg. Wineskin-2.6.2.app)

**Comment by Gcenx** [comment]
Yes Wineskin-3.0.0.app

Is the current name for my wrapper and it’s the only one in the folder.

**Comment by vitor251093** [comment]
And it's in the Wineskin wrapper folder? (~/Library/Application Support/Wineskin/Wrapper)

If so, that's weird. The image that you showed only appear if the folder does not exists or if there is no file inside the folder. There is any change that may be a hidden file in the folder? Could you check with `ls -a` inside it?

**Comment by vitor251093** [comment]
That's really strange. Could you please send me your compiled master wrapper so I can test it? My e-mail is vitormm at portingkit.com

**Comment by Gcenx** [comment]
Once I get back home yeah I will email you the master wrapper I’m using

**Comment by Gcenx** [comment]
@vitor251093 Ok I tested again with the new version of PK and my most recent commit built, it works but will throw errors about the "WineskinMenuScripts" folder and contents being missing but other then that it built a blank wrapper.

I also tested building a wrapper using PK (Skyrim), other then the above seems to be working just fine now.


I'm leaving this open for the moment incase anything else comes to mind.
